src/pages/map.tsx

```diff
@@ -75,15 +75,17 @@ const Map: React.FC<Props> = (props) => {
   };
 
   const handleSchoolSearch = async (searchTerm: string) => {
-    const response = await fetch(
-      `/api/searchSchools?searchTerm=${searchTerm}`,
-    ).then((res) => res.json() as Promise<{ schools: School[] }>);
-
-    return response.schools.map((school) => ({
-      label: school.name,
-      value: school.name,
-      item: school,
-    }));
+    const results = props.schools
```

Comment:
Fantastic, @mattgianni  ... I wondered why we didn't just .filter.map, but hadn't looked into it enough to realize that props.schools was already in component state! Great fix.

Reply:
Also avoids possible problems with hydration where the db was updated, but app not rebuilt, causing api to return schools that weren't in the hydrated list. Great fix.

Reply:
thx, ... i thought it was going to be a little tougher too, but the schools were already being pulled into the component.